.ci/job/vm/Makefile.include

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2021 Red Hat
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Specific Make targets for the VM runner.
+#
+
+KATA_TESTS_VM_ENGINE ?= vagrant
+supported_vm_engines:=$(shell ./job/vm/vm_runner.sh -e)
+
```

Comment:
You have two layers of abstraction here: first local vs. vm, then vagrant vs. other possible VM engines.  Is it worth the complexity?  Could you just have "local" and "vagrant" as the top level options.